dts/arm/st/f4/stm32f4.dtsi

```diff
@@ -428,6 +459,12 @@
 				compatible = "st,stm32-counter";
 				status = "disabled";
 			};
+
+			qdec {
+				compatible = "st,stm32-qdec";
+				status = "disabled";
+				input-filter-level = <0>;
+			};
 		};
```

Comment:
It would be nice to at least complete the addition of qdec nodes for the f4 series (f405.dtsi, f410.dtsi, f412.dtsi)

Reply:
As suggested by @str4t0m I looked more in detail on the Reference Manuals of F405, F410 and F412 and found that timers which support this quadrature encoder feature are: 1 to 5, 8.
I'll fix the code accordingly